examples-gallery/plot_crane_moving_a_load.py

```diff
@@ -0,0 +1,294 @@
+# %%
+"""
+Crane moving a load
+===================
+
+A load is moved by a crane. The load is rotably connected to the crane by a
+massless rod. The goal is to move the load from the initial position to the
+final position in the shortest possible time. The load must not 'overswing'
+the final position. The load is moved by a force applied to its
+suspension point. The jib of the crane is extended in the horizontal
+X direction.
+
+**Constants**
+
+- l: length of the rod attaching the load to the crane [m]
+- m1: mass of mover attached to the arm of the crane [kg]
```

Comment:
The variables should be latex or code:

```rst
- :math:`m_1` OR ``m1``
```

Reply:
> The variables should be latex or code:
> 
> ```
> - :math:`m_1` OR ``m1``
> ```

I had played around with :math:`.....` but did not get it to work right.
Will it only work in lines like:

"""
- :math:`....`

"""
Will it not work in lines like:
# -:math:`...`
?

Reply:
Math in RestructuredText is explained here:

https://ikerdocs-sphinx.readthedocs.io/syntax/math.html

Reply:
> Math in RestructuredText is explained here:
> 
> https://ikerdocs-sphinx.readthedocs.io/syntax/math.html

This is what I looked at, but never got it to work. I will play around some more.

Reply:
Sphinx-gallery treats the module level "docstring" as a "cell" of RestructuredText.

Reply:
This has not bee addressed.